build/debian-hyperkube-base/Makefile

```diff
@@ -19,7 +19,7 @@
 
 REGISTRY?=gcr.io/google-containers
 IMAGE?=debian-hyperkube-base
-TAG=0.6
+TAG=0.7
```

Comment:
hm, #52744 was supposed to bump this to 0.7, but it seems that got lost in a rebase. please bump this to 0.8 instead.

cc @rphillips

Reply:
Ack